docs/guide/blog.md

```diff
@@ -6,22 +6,17 @@ description: Learn module basics by writing and reading blog posts to your chain
 
 # Build a Blog
 
-Learn module basics by building a blockchain app to write and read blog posts.
+In this tutorial, you will learn how to create a blockchain with a module that lets you write to and read data from the blockchain. This module will implement create and read functionalities for a blog-like application. A user will be able to submit new blog posts and show a list of blog posts on the blockchain.
```

Comment:
hi @fadeev thanks for always making our docs better! 
- Let's use present tense for the developer who is building the blockchain, per https://developers.google.com/style/tense. 
- I went back and forth, but I in this case the end-user actions in future tense make sense. 
- We want our language to be empowering. We look to the [Digital Ocean Style Guide](https://www.digitalocean.com/community/tutorials/digitalocean-s-technical-writing-guidelines) for inspiration:

> Keep the focus on the reader and what they will accomplish. Instead of using phrases like “we will learn how to”, use phrases like “you will configure” or “you will build”. This goes a long way to motivate the reader and get them excited about your topic. In addition, keep the focus on the problem the reader is solving rather than the technology.